### ISSUE TITLE
On-demand revalidation not working on my site

### ISSUE THREAD

**Issue by pablote** [sst/open-next#147]
I'm trying to use the following feature: https://nextjs.org/docs/pages/building-your-application/data-fetching/incremental-static-regeneration#on-demand-revalidation. The revalidation of ISR routes, by means on manual on-demand API call that re renders some routes. Is this something that's supposed to work or a known missing feature?

By not working I mean, I do an API call that internally calls `res.revalidate(...)` on several routes, the API runs fine, no errors, but then when I go to the page and refresh I still see the old data, the same page that was initially deployed. What I was expecting to see was updated data.

This works well for me locally if I run the application using `npm run build && npm run start`, I can correctly see updated data after running the revalidate API call, so I seem to have implemented this correctly.

This is the revalidate API code in case it helps, this API call runs fine without errors but it doesn't seem to be doing anything:
```typescript
export default async function handler(req: NextApiRequest, res: NextApiResponse) {
  // Check for secret to confirm this is a valid request
  if (req.query.secret !== process.env.REVALIDATION_SECRET) {
    return res.status(401).json({ message: "Invalid secret" });
  }

  try {
    await res.revalidate("/products");

    let productPages = await getProductsPages();
    for (let i of range(productPages.pageCount)) {
      await res.revalidate(`/products/${i + 1}`);
    }
    return res.json({ revalidated: true });
  } catch (err) {
    return res.status(500).send({ msg: "Error revalidating", err });
  }
}
```

I've tried diagnosing the problem on AWS, but I'm not sure exactly where to look. There's a `siteServerFunction` that's logging but not showing any errors. There's also a `siteRevalidationFunction` but it doesn't seem to have logs or cloudwatch setup. If I click on "View CloudWatch Logs" then it shows an error: "The specific log group: /aws/lambda/prod-.... does not exist in this account or region", so I can't tell if this is not working or running at all. 

There's also a `siteRevalidationQueue` in SQS, looking in the monitor tab the only one going above 0 is "Number Of Empty Receives":
<img width="1302" alt="Screenshot 2023-06-27 at 7 24 40 PM" src="https://github.com/serverless-stack/open-next/assets/2463995/96330f30-79e8-432c-a72e-df02f1f84a0e">

I'm running sst@2.17.4, I've already tried removed the whole stage from AWS and redeploying from scratch, but no changes. Let me know if there's any other info I can provide, thanks.

**Comment by pablote** [comment]
Ah you're right. I can see the updated data on S3's `_cache`, I'm going to go ahead and do the CF invalidation as well, thanks for the info. BTW, why wasn't this incorporated within the "res.revalidate()" call? Isn't the idea of SST/open-next to be transparent and simply work the same way as if running from Vercel?

**Comment by khuezy** [comment]
This should be resolved in 2.1.2, please open new bug if you can't get it to work.
ref: https://discord.com/channels/983865673656705025/1147498646032617613
